doc/source/ray-core/doc_code/limit_tasks.py

```diff
@@ -1,47 +1,38 @@
-import numpy as np
-
+# __without_backpressure_start__
 import ray
 
 ray.init()
 
 
-# __defining_actor_start__
 @ray.remote
 class Actor:
-    def heavy_compute(self, large_array):
+    async def heavy_compute(self):
         # taking a long time...
+        # await asyncio.sleep(5)
         return
 
 
-# __defining_actor_end__
-
-
-# __creating_actor_start__
 actor = Actor.remote()
-# __creating_actor_end__
 
-# __executing_task_start__
+NUM_TASKS = 1000
```

Comment:
Would it be better to make the number of tasks infinite so that it better matches the issue description?

Reply:
Since the test code will be run by our CI. I couldn't figure out a good way to show infinite on the example but a small number for CI.